### DISCUSSION TITLE
c++raylib実行ファイル実行

### DISCUSSION THREAD

**Discussion by Crab-Club1** [HamsterCrab/Science-PC_Club_Game#64] [Show and tell]
#include "raylib.h"
#include <cstdlib> // system関数を使用するために必要

int main() {
    // ウィンドウの初期化
    InitWindow(800, 600, "raylib Example");

    // メインループ
    while (!WindowShouldClose()) {
        // 更新処理

        // 描画処理
        BeginDrawing();
        ClearBackground(RAYWHITE);
        DrawText("Click anywhere to run the 'inm' executable.", 100, 200, 20, LIGHTGRAY);
        EndDrawing();

        // マウスクリックを検出
        if (IsMouseButtonPressed(MOUSE_LEFT_BUTTON)) {
            // `inm` 実行ファイルを実行
            system("./inm"); // Linux または macOS の場合
            // system("inm.exe"); // Windows の場合
        }
    }

    // ウィンドウの終了処理
    CloseWindow();

    return 0;
}

**Comment by Hamster-crab** [comment]
なんで大事なところが消えてるコードを送ってくるんですか？
淫夢廚ではないのですか？
ちゃんとコードの確認をしてから行って下さい
C++好きなんじゃないのか？
